erpnext/accounts/utils.py

```diff
@@ -794,7 +794,7 @@ def get_children(doctype, parent, company, is_root=False):
 
 	return acc
 
-def create_payment_gateway_account(gateway):
+def create_payment_gateway_account(gateway, payment_channel="Email"):
```

Comment:
A patch might be needed to update payment channel in already existing payment gateway accounts otherwise this method won't get you the appropriate account.

Reply:
added changes